docs/source/plugin/bot.rst

```diff
@@ -96,6 +96,125 @@ the :meth:`~sopel.bot.SopelWrapper.action` method::
     bot.action('does something')
 
 
+Do it with style
+================
+
+.. Custom role definition to apply custom style to inline text
+
+.. role:: red
+    :class: red
+
+.. role:: boldred
+    :class: bold red
+
+.. role:: underline
+    :class: underline
+
+.. role:: strike
+    :class: strike
+
+.. role:: bolditalic
+    :class: bold italic
+
+.. role:: spoiler
+    :class: spoiler
+
+
+When the bot talks, replies, or acts, it can do so with style: colors,
```

Comment:
Silly idea, probably not worth implementing just because of the sheer length of markup it would require: use the UK spelling just this once and make it rainbow. 😜

Reply:
lol I'm sure I can do that 🌈

Reply:
Well, OK I can, but the time and effort is not worth it at the moment.

Reply:
Yup. It would take seven different roles, one for each letter. Very not worth it!